nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Core/Files/JsonBuildFile.cs

```diff
@@ -27,6 +27,18 @@ public void UpdateProperty(string[] propertyPath, string newValue)
 
     private void ResetNode()
     {
-        Node = new Lazy<JsonNode?>(() => JsonHelper.ParseNode(Contents));
+        Node = new Lazy<JsonNode?>(() =>
+        {
+            try
+            {
+                return JsonHelper.ParseNode(Contents);
+            }
+            catch (System.Text.Json.JsonException)
+            {
```

Comment:
Please is it possible to surface an error here: "Invalid configuration file". We do have this pattern in other parts of the code and I wonder if we can continue with that practice.